CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## Unrelesaed
+
+
+### Fixed
+
+- [rom-repository] Require `plugins.rb` before calling `ROM::Plugins.register` (@adam12)
+
+
 ## 5.2.4 2020-05-08
```

Comment:
@adam12 please remove this change, changelogs are automatically generated from merge commit messages so this is not needed.

Reply:
Done!